tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts

```diff
@@ -498,7 +512,8 @@ export class LineChartComponent
     this.setIsViewBoxOverridden(false);
     this.isViewBoxChanged = true;
     this.updateLineChart();
-    this.viewBoxChanged.emit(this.viewBox);
+    this.viewBoxChanged.emit(null);
+    this.userViewBox = null;
```

Comment:
Modifying an input like this is unusual and probably undesirable. It probably points to the fact that you should be relying on the redux pattern.